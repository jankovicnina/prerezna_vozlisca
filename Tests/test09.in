43 133
33 21 4.31
37 0 4.87
6 20 4.32
41 10 9.65
6 13 -0.24
41 12 5.14
8 19 1.55
27 23 4.37
9 33 8.58
23 24 7.35
11 14 3.69
3 23 4.45
37 19 -0.2
33 4 3.74
18 30 3.47
10 11 6.23
36 19 8.66
14 27 1.12
38 12 9.54
20 36 1.68
39 38 7.75
25 26 5.76
8 1 0.28
32 17 6.59
28 17 4.15
21 31 5.03
25 35 8.89
0 21 3.92
2 21 6.95
0 20 7.0
11 10 2.24
35 37 -0.15
38 7 3.59
4 3 -1.9
16 14 8.56
18 33 5.74
3 16 2.09
11 39 6.36
30 34 -1.33
0 37 3.61
24 26 -0.46
7 37 0.34
21 26 9.36
10 35 -1.24
6 42 2.93
39 10 1.93
26 22 5.24
21 35 8.95
22 25 -0.47
42 0 0.98
23 36 -1.54
37 17 2.58
28 11 3.43
41 20 8.37
5 15 -0.92
7 10 9.92
15 42 1.94
32 42 2.14
36 12 2.15
34 5 8.5
35 7 -0.16
16 8 -0.26
35 16 9.62
21 36 -1.75
26 2 7.05
36 5 5.33
1 27 3.47
16 30 0.16
32 38 8.05
2 34 -1.95
37 11 2.43
1 5 9.31
9 35 8.64
6 26 7.97
13 33 4.22
8 0 -0.7
6 9 5.67
0 34 3.91
37 40 -1.7
14 20 -0.32
23 37 0.66
42 29 8.33
29 3 0.85
1 17 7.36
38 37 1.64
10 14 8.73
29 37 8.29
32 23 9.5
32 8 6.9
40 6 2.21
10 41 0.87
37 5 -0.44
31 38 4.39
34 28 4.66
32 15 4.44
13 32 4.15
30 40 2.33
4 31 8.32
35 6 9.23
34 30 9.62
36 26 -1.89
5 2 3.97
40 8 -1.41
38 13 4.69
29 39 3.7
10 30 2.22
34 18 -0.23
12 22 3.44
39 11 -1.44
27 15 5.52
24 3 6.44
30 15 -1.61
39 27 0.92
28 35 4.08
14 42 4.86
25 16 3.35
14 38 4.13
0 41 1.5
18 29 9.86
30 19 -1.03
17 6 -1.59
21 3 6.14
40 18 8.54
4 19 1.22
32 13 2.74
30 9 5.67
19 10 1.19
33 39 7.11
39 13 8.87
26 34 7.75
15 23 5.68
4 18 2.61
19 27 4.02
