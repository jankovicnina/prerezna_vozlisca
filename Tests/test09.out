605 1092 1177 1451 1615 1671 1732 1791 1993 2792 3123 3180 3673 3836 4119 4130 4461 4494 4736 4983 5022 6366 6435 7550 7725 8019 8102 8558 8790 8815 9565 9614
